experiments/react-native-uifabric/src/hooks/RadioGroup.hooks.ts

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+//import { IKeyboardEvent } from '@office-iss/react-native-win32';
+
+export type onChangeCallback = (key: string) => void;
+
+// This hook updates the selected button and calls the client's onChange callback with the new button
+export function useAsRadioGroupSelection(defaultSelectedKey: string, userCallback?: onChangeCallback) {
+  const [selectedKey, setSelectedKey] = React.useState(defaultSelectedKey);
+
+  const onChange = React.useCallback(
+    (key: string) => {
+      setSelectedKey(key);
+      userCallback && userCallback(key);
+    },
+    [setSelectedKey, userCallback]
+  );
+
+  return { onChange, selectedKey };
+}
+
+// This hook will be used to capture onKeyDown
```

Comment:
Is this block needed? If not, delete it.

Reply:
Deleted